Documentation/JSON_Schemas/CreateGame_Response.json

```diff
@@ -0,0 +1,42 @@
+{
+  "type": "object",
+  "properties": {
+    "user": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "integer"
+        }
+      },
+      "required": [
+        "id"
+      ]
+    },
+    "game": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "integer"
+        }
+      },
+      "required": [
+        "id"
+      ]
+    }
+  },
+  "required": [
+    "user",
+    "game"
+  ]
+}
+
+Example:
+
+{
+  "user": {
```

Comment:
The example? Or returning the user ID? The idea was some sort of _very_-fatal-error chacking